README.md

```diff
@@ -2,10 +2,21 @@
 
 A module for Terasology.
 
-## Planed content
+## Getting Started
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+- You have to start the game with GooeysQuests activated and in a Perlin world.
```

Comment:
The world doesn't have to be perlin, it just needs to have substantial enough cliffs/mountains/hills to spawn the dwarf halls in.  Pelin is an example of a world generator that has these features.